src/components/Avatar/style.css

```diff
@@ -0,0 +1,42 @@
+.Avatar {
+  position: relative;
+  border-radius: 50%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  color: white;
+  background-color: var(--rvr-blue);
+  background-position: center;
+  background-size: cover;
+}
+
+.loading {
+  animation-duration: 1s;
+  animation-fill-mode: forwards;
+  animation-iteration-count: infinite;
+  animation-name: imagePlaceholderShimmer;
+  animation-timing-function: linear;
+  background: linear-gradient(to right, var(--rvr-blue) 2%, var(--rvr-blue-lite-1) 18%, var(--rvr-blue) 33%);
+  background-size: 1200px 10px;
+  position: relative;
+}
+
+.AvatarBadge {
+  position: absolute;
+  bottom: 0;
+  right: 0;
+}
+
+@keyframes imagePlaceholderShimmer {
```

Comment:
How about putting the keyframes in a global stylesheet? "Placeholder shimmer" could even be its own component.